enzy_htp/core/clusters/accre.py

```diff
@@ -42,8 +42,8 @@ class Accre(ClusterInterface):
 
     # environment presets #
     AMBER_ENV = {
-        "CPU": "source /home/shaoq1/bin/amber_env/amber22.sh",
-        "GPU": "source /home/shaoq1/bin/amber_env/amber22.sh",
+        "CPU": "source ~/setup-accre.sh",
```

Comment:
I can change it back. This is kind of my argument for a config file for a cluster not being in the actual code distribution. I changed it to work for my specific setup. That being said I also changed it so that you can alter it from the script you are calling, but I still think this is a design aspect we should change.

Reply:
Yeah I will change this. This is part of my thought that the cluster config files should not be in the distribution of EnzyHTP. I guess this is somewhat fixed since I can change the `dict` directly

Reply:
Reverted with commit 3

Reply:
Pushed commit. Marking as resolved.

Reply:
I agree that this should be set elsewhere. I will add a TODO for it. This will be part of the config for the HPC of individual user.

Reply:
Yeah I guess for ACCRE this is fine that I will open it for all ACCRE user. And it should not be changed.